libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-config.ts

```diff
@@ -1,9 +1,22 @@
 import { Type } from '@angular/core';
 
+import { SkyFilterBarFilterModalSizeType } from './filter-bar-filter-modal-size';
+
 export interface SkyFilterBarFilterModalConfig {
+  /**
+   * The modal component to display when the user selects the filter.
+   * The component should expect a `SkyFilterBarFilterModalContext` object to be provided by the modal service on instantiation.
+   * The return value of the modal save action should be a `SkyFilterBarFilterValue`.
+   * @required
+   */
   modalComponent: Type<unknown>;
-
-  modalSize?: 'small' | 'medium' | 'large' | 'full';
-
+  /**
+   * The size of the modal to display.
```

Comment:
```suggestion
   * The size of the modal to display. The valid options are `small`, `medium`, `large`, and `full`.
```
Seems like we should call out the size options like we do in the corresponding `size` description in the modal docs. That description also has a third sentence that says "This property defaults to `medium`." but I assume the "@default" here makes that unnecessary.

Reply:
After adding this suggestion, I noticed below that SkyFilterBarFilterModalSizeType also lists `full` as an option. But it's not listed as an option in the modal docs for the `size` property. Is that a bug in the modal docs? Or a discrepancy between the modal docs and filter bar docs?

Reply:
Rather than add an additional `fullScreen` property, the options for size and fullscreen are combined into one expanded option here

Reply:
Got it. I updated the suggestion to include `full` in the list of sizes.